pkg/repository/v1/sqlcv1/olap.sql

```diff
@@ -777,14 +781,16 @@ WITH locked_events AS (
         v1_tasks_olap t ON
             (dt.task_id, dt.task_inserted_at) = (t.id, t.inserted_at)
     GROUP BY
-        d.id, d.inserted_at
+        d.id, d.inserted_at, d.total_tasks
 ), updated_dags AS (
     UPDATE
         v1_dags_olap d
     SET
         readable_status = CASE
             -- If we only have queued events, we should keep the status as is
             WHEN dtc.queued_count = dtc.task_count THEN d.readable_status
+            -- If the task count is not equal to the total tasks, we should set the status to running
+            WHEN dtc.task_count != dtc.total_tasks THEN 'RUNNING'
```

Comment:
any footguns here?

Reply:
I don't think so?

Reply:
will the task count include skipped tasks / cancelled tasks? that's the only thing that comes to mind